src/Entity/DfDb.php

```diff
@@ -0,0 +1,88 @@
+<?php
```

Comment:
Mandatory changes for the validation :smile: 

```suggestion
<?php
/**
 * NOTICE OF LICENSE
 *
 * This file is licenced under the Software License Agreement.
 * With the purchase or the installation of the software in your application
 * you accept the licence agreement.
 *
 * You must not modify, adapt or create derivative works of this source code
 *
 * @author    Doofinder
 * @copyright Doofinder
 * @license   GPLv3
 */
```